drivers/timer/Kconfig.wch_ch32v00x

```diff
@@ -3,6 +3,6 @@
 
 config CH32V00X_SYSTICK
 	bool "CH32V00X systick timer"
-	depends on SOC_CH32V003 || SOC_SERIES_QINGKE_V4C
+	depends on SOC_CH32V003 || SOC_SERIES_QINGKE_V4
```

Comment:
No action required: I agree with the direction here, as the systick is part of the QingKe CPU